src/main/java/fr/pilato/elasticsearch/crawler/fs/FsCrawlerImpl.java

```diff
@@ -383,8 +385,8 @@ private void addFilesRecursively(FileAbstractor path, String filepath, Instant l
 
         // TODO Optimize it. We can probably use a search for a big array of filenames instead of
         // Searching fo 50000 files (which is somehow limited).
-        private Collection<String> getFileDirectory(String path)
-                throws Exception {
+        // --> changed to 10000 which is the max. Higher values results to a default of 10 (pagination)
```

Comment:
That's super strange as I would have expect that elasticsearch actually fail in such case.
I need to check elasticsearch code base to see if not failing is intentional.